app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java

```diff
@@ -130,10 +140,38 @@ public void bindBusinessDetail(Business business) {
             } catch (Exception e) {
                 rating.setVisibility(View.INVISIBLE);
             }
+            final String imageUrl = business.getImageUrl();
+            if (imageUrl == null || imageUrl.length() == 0){
+                Picasso.with(mContext).load("https://pyzikscott.files.wordpress.com/2016/03/yelp-placeholder.png?w=584");
```

Comment:
Static URL should be a constant so it's easily findable and can be changed if needed